README.md

```diff
@@ -21,3 +20,45 @@ Have a look at the file for details, but in short this is how it works:
 - Create a custom docker image for Drupal and nginx, and push those to a docker registry (typically that of your cloud provider).
 - Install or update our helm chart while passing our custom images as parameters.
 - The helm chart executes the usual drush deployment commands.
+
+## Local development
+
+Install [Lando](https://docs.devwithlando.io/). Add your project's name and other project-specific settings to `.lando.yml` and run `lando start`.
+
+Useful helper commands:
+
+- `lando cre` runs `composer require`
+- `lando cup` runs `composer update`
+- `lando cupdc` runs `composer update drupal/core webflo/drupal-core-require-dev --with-dependencies`
+- `lando dsi` runs `drush si --db-url=mysql://drupal8:drupal8@database:3306/drupal8`. Run `lando dsi demo_umami -y` to install `demo_umami` profile.
+
+Run `lando` for complete list of available Lando commands.
+
+### Testing
+
+Supported testing tools:
+
+#### [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) with Drupal, DrupalPractice standards
```

Comment:
We actually have our own phpcs.xml configuration, which extends the Drupal standard to make it a bit more relaxed where it makes sense.

Reply:
Any ideas of how to automatically set `installed_paths` for this config? I introduced https://github.com/DealerDirect/phpcodesniffer-composer-installer earlier and it doesn't pick up..

Reply:
Resolved with 0aae671.